site/using-smart-accounts/transfer-ownership/modular-account.md

```diff
@@ -0,0 +1,144 @@
+
+All Modular Accounts have `MultiOwnerPlugin` pre-installed upon creation, exposing following methods for account owners to update (add or remove) and read the current owners of the account:
+
+```solidity
+/// @notice Update owners of the account. Owners can update owners.
+/// @param ownersToAdd The address array of owners to be added.
+/// @param ownersToRemove The address array of owners to be removed.
+function updateOwners(address[] memory ownersToAdd, address[] memory ownersToRemove) external;
+
+/// @notice Get the owners of `account`.
+/// @param account The account to get the owners of.
+/// @return The addresses of the owners of the account.
+function ownersOf(address account) external view returns (address[] memory);
+```
+
+When you connect your Modular Account to `SmartAccountClient` you can extend the client with `multiOwnerPluginActions`, which exposes a set of methods available to call the installed `MultiOwnerPlugin` with the client connected to the account.
+
+### 1. Check if an address is one of the current owners of a Modular Account
+
+You should first extends the `SmartAcountClient` connected to a Modular Account, which already comes with `MultiOwnerPlugin` installed upon creation, with client to `multiOwnerPluginActions` for the client to include the `MultiOwnerPlugin` actions.
+
+Then, you can use the `readOwners` method of the `multiOwnerPluginActions` extended smart account client to check if a given address is one of the current owners of a Modular Account.
+
+::: code-group
+
+```ts [example.ts]
+import { smartAccountClient as modularAccountClient } from "./smartAccountClient";
+import { multiOwnerPluginActions } from "@alchemy/aa-accounts";
+
+// this will return the address of the smart account currently connected to the client
+const accountAddress = await modularAccountClient.getAddress();
+
+// [!code focus:99]
+// extend smart account client with multiOwnerPluginActions to call MultiOwnerPlugin methods
+const pluginActionExtendedClient = modularAccountClient.extend(
+  multiOwnerPluginActions
+);
+
+const ownerToCheck = "0x..."; // the address of the account to check the ownership of
+
+// returns a boolean whether an address is an owner of account or not
+const isOwner = await pluginActionExtendedClient.isOwnerOf(ownerToCheck);
+```
+
+<<< @/snippets/aa-alchemy/connected-client.ts [smartAccountClient.ts]
+
+:::
+
+### 2. Get the all current owners of a Modular Account
+
+You can use the `readOwners` method on the `multiOwnerPluginActions` extended smart account client to fetch all current owners of the connected Modular Account.
+
+::: code-group
+
+```ts [example.ts]
+import { smartAccountClient as modularAccountClient } from "./smartAccountClient";
+import { multiOwnerPluginActions } from "@alchemy/aa-accounts";
+
+// this will return the address of the smart account currently connected to the client
+const accountAddress = await modularAccountClient.getAddress();
```

Comment:
this isn't being used anywhere right?

also the method `getAddress` is not `async`
```suggestion
```